src/components/SectionList.re

```diff
@@ -195,4 +204,4 @@ external make:
     ~testID: string=?
   ) =>
   React.element =
-  "SectionList";
+  "SectionList";
```

Comment:
```suggestion
  "SectionList";

```

Reply:
On the last version there was no sectionData (that is just in master I guess). But it might be breaking in other ways?

Reply:
Yeah maybe we introduced a breaking change anyway. Not sure how breaking it was.

Reply:
Sorry, I didn't specifically point out the missing newline here.

@jfrolich is right, the previous PR was not in 0.60. Even a breaking change will be easy to handle as one simply has to remove the typecasting function.

Reply:
@jfrolich tiny thing but can you keep the new line please?

Reply:
I think refmt removes it. Is it a special refmt setting?

Reply:
Not sure, I will investigate...